Cargo.toml

```diff
@@ -52,7 +52,7 @@ path = "examples/deferred/main.rs"
 name = "flowmap"
 path = "examples/flowmap/main.rs"
 
-[[target.x86_64-pc-linux-gnu.example]]
+[[example]]
```

Comment:
I'm thinking of writing doc-tests for glfw and sdl instead. SDL needs something to test it anyway.